test/unit/spec/nbextensions/bulkImportCell/tabs/configure-spec.js

```diff
@@ -81,5 +98,85 @@ define([
                     expect(container.innerHTML).toEqual('');
                 });
         });
+
+        it('should toggle the active file type', () => {
+            initialState = {
+                state: 'editingIncomplete',
+                params: {
+                    dataType1: 'incomplete',
+                    dataType2: 'incomplete',
+                },
+                selectedFileType: 'dataType1',
+            };
+            const modelData = Object.assign({}, TestAppObject, { state: initialState });
+            ['dataType1', 'dataType2'].forEach((dataType) => {
+                modelData.app.fileParamIds[dataType] = modelData.app.fileParamIds.fastq_reads;
+                modelData.app.otherParamIds[dataType] = modelData.app.otherParamIds.fastq_reads;
+                modelData.inputs[dataType] = [];
+                modelData.params[dataType] = { filePaths: [], params: {} };
+            });
+            const _model = Props.make({
+                data: modelData,
+                onUpdate: () => {},
+            });
+            const _typesToFiles = {
+                dataType1: {
+                    appId,
+                    files: ['file1', 'file2'],
+                },
+                dataType2: {
+                    appId,
+                    files: ['file3', 'file4'],
+                },
+            };
+            const configure = ConfigureTab.make({
+                bus,
+                model: _model,
+                specs,
+                typesToFiles: _typesToFiles,
+            });
+            return configure
+                .start({
+                    node: container,
+                })
+                .then(() => {
+                    const ftPanel = container.querySelector('[data-element="filetype-panel"]');
+                    // expect that both buttons are there, and the first is selected
+                    const btn1 = ftPanel.querySelector('[data-element="dataType1"]');
+                    expect(
+                        btn1.classList.contains(
+                            'kb-bulk-import-configure__filetype_panel__filetype_button--selected'
```

Comment:
probably would have been a little easier just to make this a variable, but it doesn't really matter